src/utils/featheryClient/index.ts

```diff
@@ -530,6 +633,10 @@ export default class FeatheryClient extends IntegrationClient {
       eventData.event === 'load'
         ? eventData.previous_step_key
         : eventData.step_key;
```

Comment:
let stepKey;
if (eventData.event === 'load') {
    stepKey = eventData.previous_step_key;
} else {
    stepKey = eventData.step_key;
    this.debouncedSubmitCustom.flush();
}